kubectl-cloudflow/cmd/status.go

```diff
@@ -69,12 +69,12 @@ func calcAppStatus(applicationCR *domain.CloudflowApplication) string {
 	crashing := false
 
 	for _, d := range applicationCR.Spec.Deployments {
-		if d.Replicas == 0 && d.Runtime == "akka" {
+		if d.Replicas == nil && d.Runtime == "akka" {
 			expectedPods = expectedPods + 1 // TODO replica defaults
-		} else if d.Replicas == 0 && d.Runtime == "spark" {
+		} else if d.Replicas == nil && d.Runtime == "spark" {
 			expectedPods = expectedPods + 1 + 2 // TODO replica defaults 1 driver, 2 executors
 		} else {
-			expectedPods = expectedPods + d.Replicas
+			expectedPods = expectedPods + *d.Replicas
```

Comment:
if `d.Replicas == nil` && runtime is `flink` this will fail here.

Reply:
fixed